scanners/zap-extended/scanner/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM python:3.9.0-alpine
+COPY . /zap-extended/
+RUN pip3 install -r /zap-extended/requirements.txt
+CMD ["/bin/sh"]
```

Should ideally come with a non privilidged user by default:

```suggestion
RUN addgroup --system --gid 1001 zap-extended && adduser zap-extended --system --uid 1001 --ingroup zap-extended
USER 1001
CMD ["/bin/sh"]
```